tests/skr/docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: '3'
+services:
+  proxy:
+    image: $REGISTRY/skr/proxy:$TAG
```

Comment:
Will cleanup still take care of these images with the '/' included?

Reply:
You're right I forgot the cleanup step only cleaned up images where the tag matches the branch, I'll update it to delete all old images

Reply:
Actually, maybe it's worth waiting until all testing has moved over to being done here? That way we don't break the currently running tests in examples? I think that's worth having some slightly longer living images for now

Reply:
That works. Since the cleanup removes them all, they'll get removed when you that later.